src/dataServices/dynamoDbParamBuilder.ts

```diff
@@ -102,12 +114,25 @@ export default class DynamoDbParamBuilder {
         };
     }
 
-    static buildPutAvailableItemParam(item: any, id: string, vid: number) {
+    /**
+     * Build DDB PUT param to insert a new resource
+     * @param item
+     * @param id
+     * @param vid

```

Comment:
I would argue put in a comment or delete. item would be a good one to add but I think id & vid are self-explanatory

Reply:
deleted `id` and `vid`. Added a comment for `item`